api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -474,6 +478,18 @@ model NodePoolSpec {
   tuningConfigs?: string[];
 }
 
+/** Taint is controlling the node taint and its effects */
+model Taint {
+  /** The key of the taint */
+  key: string;
```

Comment:
There are restrictions about the accepted character sets and lengths for "key" and "value". We should document them

Reply:
I created a custom `label` scalar with imposing a limits for the string to comply with k8s rules.